modules/hcp-virt-taints-tolerations.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-prepare/hcp-distribute-workloads.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-virt-taints-tolerations_{context}"]
+= Custom taints and tolerations
+
+For {hcp} on {VirtProductName}, by default, pods for a hosted cluster tolerate the `control-plane` and `cluster` taints. However, you can also use custom taints on nodes so that hosted clusters can tolerate those taints on a per-hosted-cluster basis by setting `HostedCluster.spec.tolerations`.
+
+:FeatureName: Passing tolerations for a hosted cluster
+include::snippets/technology-preview.adoc[]
+
+.Example configuration
+[source,yaml]
+----
+  spec:
+    tolerations:
+    - effect: NoSchedule
+      key: kubernetes.io/custom
+      operator: Exists
+----
+
+You can also set tolerations on the hosted cluster while you create a cluster by using the `--tolerations` hcp CLI argument.
+
+.Example CLI argument
+[source,terminal]
+----
+--toleration="key=kubernetes.io/custom,operator=Exists,effect=NoSchedule"
+----
+
+For fine granular control of hosted cluster pod placement on a per-hosted-cluster basis, use custom tolerations with `nodeSelectors`. You can co-locate groups of hosted clusters and isolate them from other hosted clusters. You can also place hosted clusters in infra and master nodes.
```

Comment:
IMHO, we should add the note
```
Tolerance only spreads all the control-plane’s pods. The infrastructure and workload pods triggered by creating the node pool (importers, virt-launcher, etc.) are not part of the control-plane and will not be subject to spread tolerance.
```
@davidvossel @tiraboschi Could you help me check if the note I added is appropriate?

Reply:
I'd try to be more generic:
```
Tolerations set on the hosted-cluster will only spreads to the control-plane’s pods. Other pods running on the management cluster and infrastructure related pods such as the pods needed to run VMs should be eventually configured by other means.
```

Reply:
@tiraboschi @LiangquanLi930 Would this revision work?
`Tolerations on the hosted cluster spread only to the pods of the control plane. To configure other pods that run on the management cluster and infrastructure-related pods, such as the pods to run VMs, you need to use a different process.`

Reply:
looks good